src/main/java/Storage.java

```diff
@@ -0,0 +1,79 @@
+import task.*;
+
+import java.io.*;
+import java.util.Scanner;
+
+/**
+ * Storage is a utility class that handles the saving and loading of tasks for the Duke program.
+ */
+public class Storage {
+    /**
+     * Saves the a TaskList into a specified file.
+     * @param path  Path of the file.
+     * @param taskList List of tasks to be saved.
+     * @throws IOException Throws IOException when it encounters an IO error.
+     */
+    public static void saveTasks(String path, TaskList taskList) throws IOException {
```

Comment:
I like the use of a verb for this method. It helps to make the purpose of the method clearer. I also see good use of nouns and verbs for classes and methods respectively in other areas, so keep up the good work!